internal/resource/sysfs-device.go

```diff
@@ -64,3 +64,7 @@ func (d vfioDevice) IsMigEnabled() (bool, error) {
 func (d vfioDevice) IsMigCapable() (bool, error) {
 	return false, nil
 }
+
+func (d vfioDevice) GetDisplayMode() (string, error) {
+	return resolvePCIAddressToMode(d.nvidiaPCIDevice.Address)
```

Comment:
in a similar question to the nvml case, does it make sense to use the `d.nvidiaPCIDevice.` functions here directly?

Reply:
I changed the implementation here. `Class` is the property of the PCI device which is changed when the GPU is switched between modes. So we should just be getting `class` of the device. The interpretation of label based on class is the labelers job.